packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -36,23 +39,19 @@ const useLazyComponents = (componentUids = []) => {
       setStore(Object.fromEntries(componentStore));
     };
 
-    if (componentUids.length && loading) {
+    if (loading) {
```

Comment:
That's why we need to check only loading there. 
Loading will be **true** only when `!!newUids.length` so this means only when we got at least something to load.
Also it will be **false** in two cases:

1. Loading is already completed and we don't need to load it again.
2. There is no uids to load so we don't even start it.